FORMAT.md

```diff
@@ -212,6 +212,40 @@ Note: the HRDF file is ill-formed and should generate a parsing error if both `m
 <actuator type="X5-9" mass_offset="0.2"/>
 ```
 
+### `<include>`
+
+The `include` element is used to allow commonly used snippets of HRDF to be reused in a single file or in multiple files.  The `include` element is used in place of a normal robot model element, and the filename referenced must be a complete and valid .HRDF file.  The contents of the "robot" element of this .HRDF file replace the "include" element in the final HRDF.
+
+Note that any attributes on the "robot" element in the included are ignored.  A compliant parser should generate and error if the file cannot be found.
+
+
+**Required Attributes:**
+- `path` (string) Path to the HRDF file to be included. A forward slash should be used as a file separation character. Absolute paths begin with "/", while relative paths do not.  The double dot ".." pattern moves up a directory.
```

Comment:
I could be convinced to support different file separators, but I like keeping this to a single convention in the files...

Reply:
"relative" means relative to the currently parsed hrdf file

Reply:
good call -- we can update that description.